infrastructure/templates/public-api/application/shared/postgreSqlFlexibleServer.bicep

```diff
@@ -0,0 +1,62 @@
+@description('Specifies the location for all resources.')
+param location string
+
+@description('Specifies common resource prefix')
+param commonResourcePrefix string
+
+@description('Administrator login name.')
+param postgreSqlAdminName string
+
+@description('Administrator password.')
+@secure()
+param postgreSqlAdminPassword string
+
+@description('SKU name.')
+param postgreSqlSkuName string = 'Standard_B1ms'
+
+@description('Storage Size in GB.')
+param postgreSqlStorageSizeGB int = 32
+
+@description('Autogrow setting.')
+param postgreSqlAutoGrowStatus string = 'Disabled'
+
+@description('Firewall rules.')
+param postgreSqlFirewallRules {
+  name: string
+  cidr: string
+}[] = []
```

Comment:
Do a lot of the params need to be prefixed with `postgreSql`? E.g. could be `storageSizeGb` instead of `postgreSqlStorageSizeGB`.

Reply:
Sorted ta